### DISCUSSION TITLE
Where to configure trusted origins?

### DISCUSSION THREAD

**Discussion by rkade24** [goauthentik/authentik#4735] [Q&A] [answered]
Hello. I am a bit of a noob to Authentik and I've been trying to follow several guides on getting it setup. However they all seem to omit configuring trusted origins. I can get the app setup and running in Unraid just fine, get to the admin interface just fine, but when I try to configure some applications I keep getting an error like `CSRF Failed: Origin checking failed - https://my.domain does not match any trusted origins.`. All I have been able to find about this is adding the domain to the host config file but I have no idea where that is. Most unraid docker applications have configuration in appdata somewhere but Authentik seems to be setup differently from what I am used to.  There was one unaswered post about this from a year ago so I figured I'd ask again. Any help would be appreciated.

**Comment by BeryJu** [comment]
Check https://goauthentik.io/docs/troubleshooting/csrf

**Comment by rkade24** [comment]
So HTTP_HOST in my case includes a port and that particular article says to ensure that it does not, but doesn't say how to correct it if it does. How would I got about doing that?

**Reply by BeryJu** [reply]
Compare your reverse proxy config with https://goauthentik.io/docs/installation/reverse-proxy, the important part being `proxy_set_header Host $host;`

**Comment by rkade24** [comment, marked as answer]
Ok so I figured it out. I am using SWAG for my reverse proxy. There is a proxy.conf that is included in all of the baked in configurations for various apps. That was out of date, and had that port included on the host header. Updated it and it seems to be working now. Thank you!
